src/indent.js

```diff
@@ -11,15 +11,19 @@
  */
 module.exports = function rawtext() {
```

Comment:
Missed this before but this function should probably be called `indent`

Reply:
oh, good catch!